Lessonk_10 (Function_begining)/HomeWork_10_2(prime_definition).py

```diff
@@ -0,0 +1,16 @@
+print('enter the number')
+num = int(input())
+
+
+def is_prime():
+    if num >= 1000:
+        return False
+    elif num <= 1:
+        return False
+    else:
+        return True
```

Comment:
This is not a valid decision for a task
Example here https://github.com/MarinaZelenska/test_repo_python/pull/7/files#diff-3db5166f76ccde2b488756e83f1e53f967565c694e552ccb1097e4952562d601